lib/std/c/freebsd.zig

```diff
@@ -52,8 +52,7 @@ pub const pthread_rwlock_t = extern struct {
 };
 
 pub const pthread_attr_t = extern struct {
-    __size: [56]u8,
-    __align: c_long,
+    inner: ?*anyopaque = null,
```

Comment:
Why is this given a default value? Also wouldn't `pub const pthread_attr_t = ?*anyopaque;` be a more direct translation?

Reply:
I suppose so, I'm just copying what's above.

Reply:
Similar definitions are used for DragonflyBSD and OpenBSD. Perhaps it would be better to change them all in a followup commit?

Reply:
Sure, that's fine.